mail/rspamd/src/opnsense/service/templates/OPNsense/Rspamd/bad_file_extensions.map

```diff
@@ -0,0 +1,5 @@
+{% if helpers.exists('OPNsense.Rspamd.general.enabled') and OPNsense.Rspamd.general.enabled == '1' and helpers.exists('OPNsense.Rspamd.multimap.badfileextension') and OPNsense.Rspamd.multimap.badfileextension != '' %}
+{% for host in OPNsense.Rspamd.multimap.badfileextension.split(',') %}
+{{ host }}
```

Comment:
Why did you choose "host" .. wouldn't  file or extension make more sense?

Reply:
copy and paste